### PR TITLE
Run batch file for windows users

### PR DESCRIPTION
**Pull request by bharathyes**
#27 
Run batch file to support windows users without bash.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,11 +73,19 @@ Once the Excel file path is configured, run the main class `RestelApplication` t
...
-`sh scripts/run.sh  <excel file path>.`
+*nix:
+
+`sh scripts/run.sh  <excel file path>`
+
+Windows:
+
+`.\scripts\run.bat <excel file path>`
+
+This script when given your restel excel file as input will run the tests and use `allure` CLI tool to generate and display the reports in a browser. e.g.,:
```
Comment (kannangce) [inline README.md:84]:
```suggestion
This script when given your restel excel file as input will run the tests and use `allure` CLI tool to generate and serve the reports in a browser. e.g.,:
```

**Comment by kannangce** [comment]
Can we also update the Reademe in quickstart, so that Windows user's also find it helpful?

**Comment by bharathyes** [comment]
> Can we also update the Reademe in quickstart, so that Windows user's also find it helpful?

Changed this file to .md and have followed the restel-docs but added new section for windows. Check the new PR in restel-docs and advice on the best approach.

File: quickstart/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# About
+
+The below steps would help you get a feel of Restel by running a demo server and test it with a pre-written excel file.
+
+## Pre-requisite
+
+- Docker
+- [JsonBox](https://github.com/vasanthv/jsonbox)
+
+> Jsonbox, if started with docker compose, will run mongo. If facing issues, run mongo manually and start Jsonbox with `npm start`
```
Comment (kannangce) [inline quickstart/README.md:10]: This can be removed from prerequisite.

File: quickstart/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+### *nix users
+
+1. Go to the root folder of the project and run `make setup` to install the dependencies for reporting in your machine.
+2. Start [Jsonbox](https://github.com/vasanthv/jsonbox) by running `docker-compose up`.
+3. To execute the test, go to the home folder and run, `make demo-run`.
+
+### Windows users 
+1. Manually install JDK11 and [allure](https://docs.qameta.io/allure/#_installing_a_commandline)
+    * If you have [scoop](https://scoop.sh/) package manager, run `scoop install allure`.
+2. Start [Jsonbox](https://github.com/vasanthv/jsonbox) by running `docker-compose up`
```
Comment (kannangce) [inline quickstart/README.md:25]:
```suggestion
2. Start [Jsonbox](https://github.com/vasanthv/jsonbox) by running `docker-compose up`. If `docker-compose` is not working, please refer to the [instruction](https://github.com/vasanthv/jsonbox#how-to-run-locally) in jsonbox to have it up & running.
```

**Review by kannangce — APPROVED** [review]

**Pull request merged**